src/sentry/workflow_engine/endpoints/organization_detector_details.py

```diff
@@ -201,6 +203,10 @@ def delete(self, request: Request, organization: Organization, detector: Detecto
 
         RegionScheduledDeletion.schedule(detector, days=0, actor=request.user)
         detector.update(status=ObjectStatus.PENDING_DELETION)
+
+        if detector.type == MetricIssue.slug:
+            schedule_update_project_config(detector)
```

Comment:
I don't love special casing this but since deletion doesn't happen in any of the detectors this seems the simplest way